### ISSUE TITLE
Documentation for disabled_rules in .editorconfig is outdated (as it should be ktlint_disabled_rules)

### ISSUE THREAD

**Issue by carstenhag** [pinterest/ktlint#1674]
## Expected Behavior
Documentation does not mention deprecated properties. It should mention `ktlint_disabled_rules`.

## Observed Behavior
But it does:
- https://pinterest.github.io/ktlint/rules/configuration-ktlint/#disabled-rules
- https://pinterest.github.io/ktlint/faq/#how-do-i-globally-disable-a-rule

## Steps to Reproduce
Use `disabled_rules`, get `15:19:37.547 [pool-1-thread-10] WARN com.pinterest.ktlint.core.api.UsesEditorConfigProperties - Property 'disabled_rules' is deprecated: Rename property 'disabled_rules' to 'ktlint_disabled_rules' in all '.editorconfig' files.
`

## Your Environment

ktlint 0.47.0

```
[*.{kt,kts}]
disabled_rules=trailing-comma-on-declaration-site
```

**Comment by paul-dingemans** [comment]
Fixed via #1671

**Comment by carstenhag** [comment]
Thanks!
